website/frontend/src/components/blocks/ProgramDetails/ProgramDetails.tsx

```diff
@@ -115,6 +116,32 @@ export const ProgramDetails: VFC<Props> = ({ setDroppedFile, droppedFile }) => {
     }
   };
 
+  const calculateGas = async (values: any, setFieldValue: any) => {
```

Comment:
I guess we can specify types

![image](https://user-images.githubusercontent.com/67055679/153393457-5b324057-84aa-472e-8519-fb0313422d61.png)
![image](https://user-images.githubusercontent.com/67055679/153393532-42ae854b-c2f7-4b3f-8856-cc9fb55ffeeb.png)